### DISCUSSION TITLE
Compatibility issue with new Recaptcha Enterprise?

### DISCUSSION THREAD

**Discussion by rgbjay** [solspace/craft-freeform#143] [Feature Requests]
Hi,

I am just trying to setup Recaptcha for my forms and the plugin asks for the Secret Key even though Google no longer generates one for you, just the key ID. Have I missed something or is this a bug?

Thanks in advance

**Steps to reproduce**

1. Enable Recaptcha in FreeForm settings
2. Enter the reCAPTCHA Site Key value
3. reCAPTCHA Secret Key is no longer provided by Google from what I can see

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.6.12.1
- Freeform Version: 3.11.3
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh

**Comment by gustavs-gutmanis** [comment]
I don't think it's possible to validate a posted recaptcha payload without a secret.

If you go to https://www.google.com/recaptcha/ and click on `Admin Console`, then go to the settings by pressing the cog icon, you should be able to see `reCAPTCHA keys` link/button, which, when clicked - will show you both the site key and secret.

**Comment by kjmartens** [comment]
@rgbjay Just rereading this one, we realize you're referring to the new Enterprise edition of reCAPTCHA. This is currently not supported, but we'll look into adding support for it in the near future. In the meantime, you can continue to use the free v2/v3 options of reCAPTCHA until we have support for it. :)

**Comment by kjmartens** [comment]
This was tested and verified working with Freeform 5.4. 🙂
